examples/systemd/sidekiq.service

```diff
@@ -42,13 +42,19 @@ WatchdogSec=10
 
 WorkingDirectory=/opt/myapp/current
 # If you use rbenv:
-# ExecStart=/bin/bash -lc 'exec /home/deploy/.rbenv/shims/bundle exec sidekiq -e production'
+# ExecStart=/bin/bash -lc 'exec /home/deploy/.rbenv/shims/bundle exec sidekiq -e production' -C config/sidekiq.yml
```

Comment:
This is the default so there is no need to explicitly configure it.

Reply:
By default does it use the action mailer queue? I added this because I had to set up an extra queue on the sidekiq with the name "mailers" so that I can make the system for sending emails using the method (deliver_later). Without it, he only creates the queue with the name "default", if I'm not mistaken.

My file sidekiq.yml

My file sidekiq.yml:

`:concurrency: 1
:queues:
  - default
  - mailers`

Reply:
I found it interesting to leave this information in the service example to better guide people. In my case I was a little lost to solve the problem using the gem 'capistrano/sidekiq'. I believe that if you leave this information visible here, people will not have to go through the same problem that and many other people have already.

Reply:
https://stackoverflow.com/a/28504367 How to set the AM queue to “default”.